### DISCUSSION TITLE
Watch and Serve when using Programmatic API

### DISCUSSION THREAD

**Discussion by gianlucaparadise** [11ty/eleventy#2396] [Q&A]
Hello 👋 

I'm using the Programmatic API to build my templates and it is working correctly. This means that I do not use `npx @11ty/eleventy --input=src` to build, but I use my custom script. What if I want to watch and serve? The `npx @11ty/eleventy --input=src --serve` command is using its own build process.

Thanks!

**Comment by earlAchromatic** [comment]
@gianlucaparadise did you ever figure this one out?
